### PR TITLE
fix(stdlib/rows): add colums to row response

### PR DESCRIPTION
**Pull request by arriqaaq**
- adds columns from SQLQueryResult to row response
- columns from rows now fetched from columns field, not rows
- add tests for connection (Conn) with actual db connection
- add test for empty table for QueryContext in a connection

References: #1288

### CODE REVIEW

File: pkg/stdlib/rows_test.go [hunk 1]
```diff
@@ -35,6 +35,11 @@ func TestRows(t *testing.T) {
...
 			Values:  []*schema.SQLValue{{Value: nil}},
 		}},
+		columns: []*schema.Column{
+			{
+				Name: "c1",
+			},
+		},
 	}
 
 	ast := r.Columns()
```
Comment (jeroiraz) [inline pkg/stdlib/rows_test.go:45]: it would be good having a test case with empty rows and still returning the list of columns

File: pkg/stdlib/connection_test.go [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+	rows, err := c.QueryContext(ctx, "SELECT * FROM emptytable limit 1", nil)
+	require.NoError(t, err)
+	defer rows.Close()
+
+	cols := rows.Columns()
+	require.Equal(t, len(cols), 6)
+}
+
+/*func TestConn_Ping(t *testing.T) {
```
Comment (jeroiraz) [inline pkg/stdlib/connection_test.go:194]: is this test case not working?

File: pkg/stdlib/connection_test.go [hunk 3]
```diff
@@ -0,0 +1,222 @@
+package stdlib
```
Comment (jeroiraz) [inline pkg/stdlib/connection_test.go:1]: please include the copyright notice

File: pkg/stdlib/rows_test.go (hunk 1 above)
Comment (arriqaaq) [reply pkg/stdlib/rows_test.go:45]: ack

File: pkg/stdlib/connection_test.go (hunk 2 above)
Comment (arriqaaq) [reply pkg/stdlib/connection_test.go:194]: no it doesn't, it was commented out before, so did not want to address the issue in this PR

**Review by jeroiraz — DISMISSED** [review]

**Review by jeroiraz — APPROVED** [review]
